res/database/Default/user/therob.xml

```diff
@@ -906,7 +906,7 @@
 				<member index="1" function="2">Per_custom_Freddy_429</member>
 			</staff>
 			<groups target_groups="64" pro_pressure_groups="0" contra_pressure_groups="0" />
-			<data country="UDSSR" year="1964" distribution="2" maingenre="11" subgenre="" flags="0" blocks="2" price_mod="1.00" />
+			<data country="RU" year="1964" distribution="2" maingenre="11" subgenre="" flags="0" blocks="2" price_mod="1.00" />
```

Comment:
Ich wuerde sagen, UDSSR hatte "SU" (Soviet Union). RU waere Russland - was 1964 nicht so existent war.

Reply:
https://en.wikipedia.org/wiki/International_vehicle_registration_code

Reply:
OK. Ich werde die RU-Einträge nochmal durchgehen und je nach Erscheinungsjahr SU verwenden (wahrscheinlich in allen Fällen).